internal/proposals/logs/dynamic-backend-configuration/custom-operator/dynamic-fluentbit-config.md

```diff
@@ -0,0 +1,148 @@
+# Dynamic Fluent Bit Configuration
+
+This document investigates a valid design for a custom operator, which is needed to enable a dynamic in-cluster logging configuration with `Fluent Bit`. The regarding issue can be found [here](https://github.com/kyma-project/kyma/issues/11105).
+
+## Criteria
+- Customers can place fluent-bit config snippets as k8s resource in a specific or in any namespace.
+- Have in mind that the central configuration can be overwritten at any time by reconciliation, which shouldn't reset customers configuration.
+- A new config gets picked up dynamically without any further interaction (like pod restart by the user).
+- Have basic validation to give early feedback on errors -> Debugging a startup problem.
+- OpenTelemetry collector has the same problem to solve thus this proposal should be extensible.
+- There's a way to provide the auth details for a backend config in a secure way.
+
+## Proposal
+
+### Architecture
+
+To build the operator, we use an SDK for a quick and easy start-up, besides various other reasons. For this, we decided to use [`kubebuilder`](https://github.com/kubernetes-sigs/kubebuilder). `kubebuilder` is an upstream project of `Kubernetes`. It provides us with features we need for our operator with the smallest overhead. In addition, this SDK is also used in other parts of Kyma, and thus we've a more consistent picture.
+
+An alternative for `kubebuilder` would be the [`Operator-SDK`](https://github.com/operator-framework/operator-sdk) from `Red Hat`. This has similar features to the kubebuilder, with some additions like the integration of [`operatorhub`](https://operatorhub.io/) - but such features aren't needed for our purpose. Furthermore, this isn't an upstream project of Kubernetes.
```

Comment:
```suggestion
An alternative for `kubebuilder` is the [`Operator-SDK`](https://github.com/operator-framework/operator-sdk) from `Red Hat`. `Operator-SDK` has similar features to the `kubebuilder`, with some additions like the integration of [`operatorhub`](https://operatorhub.io/). We decided against it because such features aren't needed for our purpose, and because it isn't an upstream project of Kubernetes.
```